src/backend/utils/wiki-feed-parser.js

```diff
@@ -17,8 +17,8 @@ module.exports.getData = function() {
       const dom = new JSDOM(data);
       return dom.window.document.querySelector('pre').textContent.split(/\r\n|\r|\n/);
     })
-    .catch(err => {
-      throw err;
+    .catch(error => {
```

Comment:
This `catch` is not necessary.  While you're in here modifying things, can you just remove it please?